lib/v2/zhihu/xhu/auth.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const config = require('@/config').value;
+
+function reset() {
+    config.xhu = {
+        cookie: undefined,
+        udid: '',
+        sn: '',
+        access_token: '',
+        count: 0,
+    };
```

Comment:
Please use the [cache](https://docs.rsshub.app/joinus/advanced/use-cache) instead.